spec/web/config/routes_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+RSpec.describe Web.routes do
+  describe 'DELETE session' do
+    it 'has a generated path' do
+      generated_path = described_class.path(:logout)
```

Comment:
Just a style point really but what would you think to making these variable assignments lets instead, to clean up the it blocks?

Reply:
I've heard good arguments for using each of these styles. I would argue that in this case - given that the generated path is only used within this assertion - it's marginally more readable to keep the contextual setup within the same block as the expectation. For anyone scanning the specs it means they don't have to look elsewhere for context. However I don't have strong feelings towards this really so happy to change. As a general point though I think this is the kind of thing we should be quite flexible on - I.E. things that are purely stylistic.

Reply:
I'm quite keen to at least initially talk about everything in this much detail, even just stylistic stuff. More than anything it's an opportunity to look at things in a slow way that you don't get to in a fast-paced work environment. That being said lets just leave it then.